### PR TITLE
Add removeUserFromOrganization

### PR DESCRIPTION
**Pull request by sheldonvaughn**
## Description
Adds [Organization Membership functions](https://workos.com/docs/reference/user-management/organization-membership) 

- [Add User to Organization ](https://workos.com/docs/reference/user-management/organization-membership/add-to-organization)
Adds an unmanaged [User](https://workos.com/docs/reference/user-management/user) as a member of the given [Organization](https://workos.com/docs/reference/organization).

- [Remove User From Organization](https://workos.com/docs/reference/user-management/organization-membership/remove-from-organization)
Removes an unmanaged [User](https://workos.com/docs/reference/user-management/user) from the given [Organization](https://workos.com/docs/reference/organization).

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[x ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Comment by marktran** [comment]
@sheldonvaughn Can you add some details to the description since this is a public repository?

File: lib/Resource/User.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace WorkOS\Resource;
+
+/**
+ * Class AuthenticationFactorSms.
```
Comment (maxdeviant) [inline lib/Resource/User.php:6]:
```suggestion
 * Class User.
```

File: lib/Resource/User.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ */
+class User extends BaseWorkOSResource
+{
+    public const RESOURCE_TYPE = "authentication_factor";
```
Comment (maxdeviant) [inline lib/Resource/User.php:10]:
```suggestion
    public const RESOURCE_TYPE = "user";
```

File: tests/WorkOS/DirectorySyncTest.php [hunk 3]
```diff
@@ -361,7 +361,7 @@ private function usersResponseFixture()
                     ],
                     "raw_attributes" => [
                         "schemas" => ["urn:scim:schemas:core:1.0"],
-                        "name"=> [
+                        "name" => [
```
Comment (willmanduffy) [inline tests/WorkOS/DirectorySyncTest.php:364]: Let's revert the changes to this file since they are unrelated to our other changes.

File: tests/WorkOS/MFATest.php [hunk 4]
```diff
@@ -108,7 +108,7 @@ public function testVerifyFactor()
     {
         $authenticationChallengeId = "auth_challenge_01FXNX3BTZPPJVKF65NNWGRHZJ";
         $path = "auth/challenges/{$authenticationChallengeId}/verify";
-        $code ="123456";
+        $code = "123456";
```
Comment (willmanduffy) [inline tests/WorkOS/MFATest.php:111]: Let's revert these changes as well.

File: lib/Users.php [hunk 5]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace WorkOS;
+
+/**
+ * Class Users
+ */
+class Users
```
Comment (willmanduffy) [inline lib/Users.php:8]: It felt weird calling this `Users` in Ruby for some reason so I called it `UserManagement`. ymmv

**Review by willmanduffy — APPROVED** [review]
